README.md

```diff
@@ -1,128 +1,151 @@
-# Getting started with AWS UI Components
-
+# AWS-UI is now Cloudscape Design System
+ 
+On July 19th, 2022, we launched [Cloudscape Design System](https://cloudscape.design). Cloudscape is an evolution of AWS-UI. It consists user interface guidelines, front-end components, design resources, and development tools for building intuitive, engaging, and inclusive user experiences at scale. As part of the release, we have begun to publish npm packages to the new `@cloudscape-design` namespace.
+ 
+**We recommend migrating your existing AWS-UI applications to *[Cloudscape](https://cloudscape.design)*.**
```

Comment:
This sentence might raise two questions:
- What is the impact for customers who don't do it?
- How urgent is it to migrate?
 
I'd propose to add something like:
> AWS-UI will not receive any updates, starting from [date/now]. Bug fixes and new features will only be available in Cloudscape.